bitnami/cassandra/templates/statefulset.yaml

```
@@ -34,34 +42,36 @@ spec:
     spec:
 {{- include "cassandra.imagePullSecrets" . | nindent 6 }}
```

Could this line be indented? It looks a bit strange to have a non-indented line when we're adding it for all of them.
```suggestion
     {{- include "cassandra.imagePullSecrets" . | nindent 6 }}
```